pyproject.toml

```
@@ -66,4 +67,10 @@ doc = [
 ]
 
 [tool.setuptools.packages.find]
-include = ['osier']
\ No newline at end of file
+include = ['osier']
+
+[tool.pytest.ini_options]
+addopts = "--cov"
```

```suggestion
addopts = "--cov=osier tests"
```

`--cov` is checking the coverage for the tests themselves which inflates the actual coverage.

---

```suggestion
addopts = "--cov=osier"
testpaths = ["tests"]
```
Looking at the `pytest` [documentation](https://docs.pytest.org/en/6.2.x/customize.html) I'm not actually sure which one of these is correct. If the first one works then bang on! Otherwise, this one should work.